client/app/components/QueryEditor.jsx

```diff
@@ -129,6 +129,7 @@ class QueryEditor extends React.Component {
 
   onLoad = (editor) => {
     // Release Cmd/Ctrl+L to the browser
+    this.editor = editor;
```

Comment:
No need to rely on this. Can access the editor with:
```js
const { editor } = this.refEditor.current;
```

Reply:
Hm, `this.refEditor` doesn't seem to be assigned. I can't find it in code and the browser throws an error when trying to access it.

Reply:
Whoops, I dunno how I missed it..
Create the ref in the class constructor and you're good to go.

```jsx
class QueryEditor extends React.Component {
  static propTypes = {
    …
  }

  static defaultProps = {
    …
  }

  constructor(props) {
    super(props);

    this.refEditor = React.createRef(); // <-- right here
   
    …
```